### PR TITLE
(FM-4046) Update LICENSE file and misc modulesync_config changes

### PR DESCRIPTION
**Pull request by DavidS**
This doesn't change the actually license, but only updates the files to be more in line with ASF's guidelines to applying the license.

### CODE REVIEW

**Comment by DavidS** [comment]
Please do not merge!

**Comment by tphoney** [comment]
With #388 being merged i am closing this.

File: LICENSE [hunk 1]
```diff
@@ -1,14 +1,202 @@
-   Copyright 2012 R.I.Pienaar
```
Comment (igalic) [inline LICENSE]: oh?

Comment (DavidS) [inline LICENSE]: See below: https://github.com/puppetlabs/puppetlabs-concat/pull/378/files#diff-aa14599dc3fdfa9cc81a7c3c247b77b9

This change is to better align with ASF's licence application guidelines.

**Pull request closed without merging**